### PR TITLE
Fixing log4j configuration for analytics (#731)

### PR DESCRIPTION
**Pull request by pmauduit**
- removes useless console appender
- WARN debug level by default

Manually tested at compilation & runtime level on my dev instance

### CODE REVIEW

File: config/defaults/analytics/WEB-INF/classes/log4j.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 log4j.rootLogger=@shared.default.log.level@, R
 
-log4j.logger.org.georchestra.analytics=DEBUG, console
+log4j.logger.org.georchestra.analytics=WARN, R
```
Comment (fvanderbiest) [inline config/defaults/analytics/WEB-INF/classes/log4j.properties]: Should be @shared.default.log.level@ as for https://github.com/georchestra/georchestra/blob/14.06/config/defaults/catalogapp/WEB-INF/classes/log4j.properties#L3

**Comment by pmauduit** [comment]
Done

**Comment by fvanderbiest** [comment]
hmmm, should be fixed on 14.01, not on master.

**Comment by pmauduit** [comment]
:(

**Comment by pmauduit** [comment]
https://github.com/georchestra/georchestra/pull/864

**Pull request closed without merging**